homeassistant/bootstrap.py

```diff
@@ -118,7 +118,7 @@ def _setup_component(hass: core.HomeAssistant, domain: str, config) -> bool:
 
         # Assumption: if a component does not depend on groups
         # it communicates with devices
-        if group.DOMAIN not in getattr(component, 'DEPENDENCIES', []):
+        if 'group' not in getattr(component, 'DEPENDENCIES', []):
```

Comment:
This would be the alternative, not sure if prettier?

```python
group = loader.get_component('group')

if group.DOMAIN not in getattr(component, 'DEPENDENCIES', []):
    …
```